From fd72eaab9194848393e293d0f7d66774487ca379 Mon Sep 17 00:00:00 2001
From: JP Sugarbroad <jpsugar@google.com>
Date: Thu, 19 Jul 2018 17:06:15 -0700
Subject: [PATCH 14/81] [RESTRICT AUTOMERGE] Add new Android.mk to handle repo
 move

We're moving the platform/build repository down a level, then symlinking
the directories and necessary files back into build/. So if we're still
in build/, keep searching for Android.mk files, otherwise stop, since
they'll be found through the symlinks.

Bug: 109817231
Change-Id: I057957e9b201c14dee185f9c00df157f178d8a31
---
 Android.mk | 8 ++++++++
 1 file changed, 8 insertions(+)
 create mode 100644 Android.mk

diff --git a/Android.mk b/Android.mk
new file mode 100644
index 000000000..a1455d465
--- /dev/null
+++ b/Android.mk
@@ -0,0 +1,8 @@
+LOCAL_PATH := $(call my-dir)
+
+# We're relocating the build project to a subdirectory, then using symlinks
+# to expose the subdirectories where they used to be. If the manifest hasn't
+# been updated, we need to include all the subdirectories.
+ifeq ($(LOCAL_PATH),build)
+include $(call first-makefiles-under,$(LOCAL_PATH))
+endif
-- 
2.17.1

